examples/bindings/QuickstartSparseString/Program.cs

```diff
@@ -1,340 +0,0 @@
-using System;
-using System.IO;
-using System.Runtime.InteropServices;
-using System.Text;
-using TileDB.CSharp;
-using TileDB.CSharp.Examples;
-using TileDB.Interop;
-
-namespace QuickstartSparseString
```

Comment:
Is this removed because of the change to `internal` linkage for the generated classes? How could one write a demo/test against the generated C API wrapper code now?

Reply:
By directly importing the generated sources in that demo/test project. This can be done by adding something like the following lines in the project file:

```xml
<ItemGroup>
  <Compile Include="sources/TileDB.CSharp/Interop/*.cs" />
</ItemGroup>
```